libs/designer-ui/src/lib/dynamicallyaddedparameter/helper.ts

```diff
@@ -175,11 +201,25 @@ export function serialize(props: DynamicallyAddedParameterProps[]): ValueSegment
       return { ...resultPropertiesObj, [schemaKey]: propertyValue };
     }, {});
 
-  const rootObject = {
-    type: 'object',
-    properties,
-    required: requiredArray,
-  };
+  let rootObject: any;
+  if (isManualTrigger) {
```

Comment:
shouldn't this be the other way around?

Reply:
defaulting now to false, so it is now correct.